Municipality/dummy.pl

```diff
@@ -1,12 +1,18 @@
 %Definition for beliefs
-:- dynamic settings/1,
-	functions/1,
-	buildings/1,
-	indicator/3,
-	indicatorLink/2,
-	stakeholder/4,
-	zone/5,
-	building/6.
+:- dynamic stakeholders/1.
+:- dynamic settings/1.
+:- dynamic functions/1.
+:- dynamic buildings/1.
+:- dynamic self/1.
+:- dynamic indicator/3.
+:- dynamic indicatorLink/2.
+:- dynamic zone_link/4.
+:- dynamic greenId/1.
+:- dynamic stakeholder/4.
+:- dynamic zone/5.
+:- dynamic building/6.
 
```

Comment:
Change this to one dynamic block. Otherwise it will expand over multiple lines in the GOAL debugger which is very unneeded.